spec/models/acts_as_taggable_on_tag_spec.rb

```diff
@@ -1,45 +1,45 @@
 require 'spec_helper'
 
 describe ActsAsTaggableOn::Tag, :type => :model do
+  subject(:tag) { ActsAsTaggableOn::Tag }
+
   describe '.autocomplete' do
-    before do
-      @tag = ActsAsTaggableOn::Tag.create(:name => "cats")
-    end
-    it 'downcases the tag name' do
-      expect(ActsAsTaggableOn::Tag.autocomplete("CATS")).to eq([@tag])
+    let!(:tag_cats) { tag.create(name: "cats") }
 
+    it 'downcases the tag name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.